wishlist/wishlist.css

```diff
@@ -0,0 +1,209 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    --product-name: #ff2020;
+    --primary-color: #e72744;
+    --secondary-color: #F05524;
+    --primary-btn: #6495ED;
+}
+
+
+/* nav bar */
+
+.landing-page-nav {
+    display: flex;
+    justify-content: space-between;
+    background-color: #f1f1f1;
```

Comment:
try adding color variable so you can reuse it

Reply:
thanks somesh will work on it